client/scss/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function color($name) {
```

Comment:
It's a shortcut for readability. I've got `color()` all over my code. I can remove it if you want, but personally I find SCSS much harder to skim when it's got `oColorsGetPaletteColor()` all over the place.